### PR TITLE
Feature/car 2243  update boring ssl

### PR DESCRIPTION
**Pull request by sumitkandpal**
Please do not send pull requests to the BoringSSL repository.

We do, however, take contributions gladly.

See https://boringssl.googlesource.com/boringssl/+/master/CONTRIBUTING.md

Thanks!

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -667,7 +667,10 @@ file(STRINGS util/go_tests.txt GO_TESTS)
 set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS
              util/go_tests.txt)
 
-add_custom_target(
+# airtime: start
+if (NOT ANDROID)
+  add_custom_target(
```
Comment (artasano) [inline CMakeLists.txt:672]: I'm confused. The opening parenthesis is here in airtime tag but what about the closing parenthesis?

**Comment by MattMacGregor** [comment]
Wasn't the plan to keep the upstream commits unchanged when merging? I'm only seeing two commits, with one containing all the upstream changes. Won't this cause the same issue we ran into next time we attempt to update boringssl with upstream changes?

**Comment by sumitkandpal** [comment]
> Wasn't the plan to keep the upstream commits unchanged when merging? I'm only seeing two commits, with one containing all the upstream changes. Won't this cause the same issue we ran into next time we attempt to update boringssl with upstream changes?

thats a good catch. looks like i did not do the merge commit properly at the end. will create a new pr

File: CMakeLists.txt (hunk 1 above)
Comment (sumitkandpal) [reply CMakeLists.txt:672]: this is indeed  confusing, I took the block present in develop and intention was not to run these tests in android. i think it should have been like below

```
# airtime: start
if (NOT ANDROID)
# airtime: end

  add_custom_target(
    run_tests
    COMMAND ${GO_EXECUTABLE} test ${GO_TESTS}
    COMMAND ${GO_EXECUTABLE} run util/all_tests.go -build-dir
            ${CMAKE_CURRENT_BINARY_DIR}
    COMMAND cd ssl/test/runner &&
            ${GO_EXECUTABLE} test -shim-path $<TARGET_FILE:bssl_shim>
              ${HANDSHAKER_ARGS} ${RUNNER_ARGS}
    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
    DEPENDS all_tests bssl_shim handshaker fips_specific_tests_if_any
    USES_TERMINAL)
# airtime: start
endif()  
# airtime: end

```

**Comment by sumitkandpal** [comment]
closing creating a new one which shows the commit from updtream

**Pull request closed without merging**